app/src/ui/release-notes/release-notes-dialog.tsx

```diff
@@ -33,38 +36,58 @@ function accountUrl(name: string): string {
   return `https://github.com/${name.substr(1)}`
 }
 
+function linkifyIssues(
+  issueNumberLine: string
+): ReadonlyArray<JSX.Element | string> | null {
+  const trimmed = issueNumberLine.trim()
+  const issueNumbers = trimmed.split(' ')
+
+  if (issueNumbers.length === 0) {
+    return null
+  }
+
+  const linkifiedIssueNumbers = issueNumbers.map((issueNumber, index) => {
+    return (
+      <LinkButton key={index} uri={desktopIssueUrl(issueNumber)}>
```

Comment:
Could we do something similar to

```ts
return (
  <>
    {index === 0 ? null : ' '}
    <LinkButton key={index} uri={desktopIssueUrl(issueNumber)}>{issueNumber}</LinkButton>
  </>
)
```

And thus avoid the `join` method altogether?